rotkehlchen/serialization/deserialize.py

```diff
@@ -44,7 +44,7 @@ def deserialize_fee(fee: Optional[str]) -> Fee:
 
     Can throw DeserializationError if the fee is not as expected
     """
-    if fee is None:
+    if fee is None or fee == 'null':
```

Comment:
Hmm I don't like this. The string here should only be a number. Can you handle the extra `'null'` case in the single source case where you encountered it?